source/Octopus.Server.Client/Model/Endpoints/UpgradeStatusResource.cs

```diff
@@ -0,0 +1,15 @@
+using Octopus.TinyTypes;
+
+namespace Octopus.Client.Model.Endpoints;
+
+public class UpgradeStatusResource : CaseInsensitiveStringTinyType
```

Comment:
My understanding is that it's bad form to use an Enum on a public API so I've used a TinyType instead. Is this ok?